packages/next/src/lib/eslint/getESLintPromptValues.ts

```diff
@@ -0,0 +1,31 @@
+import findUp from 'next/dist/compiled/find-up'
+
+export const getESLintStrictValue = async (cwd: string) => {
+  const hasTSConfig = !!(await findUp('tsconfig.json'), { cwd })
+
+  return {
+    title: 'Strict',
+    recommended: true,
+    config: {
+      extends: hasTSConfig
+        ? ['next/core-web-vitals', 'next/typescript']
```

Comment:
@eps1lon threading https://github.com/vercel/next.js/pull/52845#issuecomment-2090036184: ACK that this is now only traditional lint rules. No more type-checked ones. 👍